src/server/string_family.cc

```diff
@@ -536,6 +547,14 @@ void StringFamily::Set(CmdArgList args, ConnectionContext* cntx) {
       if (int_arg >= kMaxExpireDeadlineSec * 1000) {
         return builder->SendError(InvalidExpireTime("set"));
       }
+
+      // Replicate any expiration argument as PXAT.
+      // TODO: Skip this flow is journal is disabled?
+      {
+        auto abs_time_str = absl::StrCat(abs_int_arg_ms.value_or(TtlToAbsExpiry(int_arg)));
```

Comment:
To me it makes more sense to use the function RecordJournal inside SetCmd::Set
because now I see this is relevant for the flow of the SET command and also the SETEX command

Reply:
We can make the RecordJournal function store the command, not call the write to journal.
Drop the NO_AUTOJOURNAL command op and use the flow of LogAutoJournalOnShard which will write the command to journal

Reply:
I suggest doing it the other way - do not use the stash here (and at all)

We cannot fully remove NO_AUTOJOURNAL for all commands in  a simple way, because its really inefficient to store for example all the deleted keys in SPOP, or storing both rewrites for the MOVE command in some kind of shard local storage

If there is a reason to, we can switch to plain rewrites in the future.